skimage/segmentation/_expand_labels.py

```diff
@@ -0,0 +1,107 @@
+""" 
+expand_labels is  inspired by code in CellProfiler, 
```

Comment:
All licensing information should please go into LICENSE.txt.

Reply:
Including this in the file was suggested by @jni who pointed out that this kind of note is also present in e.g. `_canny.py`, `_edges.py`, `_watershed.py` and a number of other algorithms that were also originally implemented for CellProfiler.

Without digging too deep License.txt seems to contain copyright notices for whole files that have benn vendored?

I can move the note elsewhere (License.txt if that is what the core team agrees on), but I'd rather err on the side of attributing the original authors one time more than necessary.

Reply:
I removed the notice from `_expand_labels.py` for now but reworded the attribution more clearly in the Notes section. We probably still need to retain the original string somewhere.
Maybe all the different Cellprofiler contributions could be acknowledged in a bundled form in License.txt for consistency. I'm not sure whether that is in agreement with the actual terms of the BSD-3 license. If so, this would be a refactor that should probably be in a separate PR anyway (and probably with input from the Cellprofiler team to make sure they are happy).

Reply:
I wasn't sure whether "inspired" meant derived from.  We have to decide if this is a copyright statement or not, because that influences under which license we release the file (CallProfiler's, or ours).

Reply:
I had changed that several times before. The core approach is based on the lines following this: https://github.com/CellProfiler/CellProfiler/blob/082930ea95add7b72243a4fa3d39ae5145995e9c/cellprofiler/modules/identifysecondaryobjects.py#L559 however, the final result has quite a few differences most importantly the arbitrary dimensions. 
I definitely want to put it back in _somewhere_ , but still don't know whether it should go at the top of the file itself (like in the other examples) or into License.txt as you suggested earlier.
Maybe "inspired by" should be "derived from original code by ...".